src/test/java/com/wikia/webdriver/common/testnglisteners/InvokeMethodAdapter.java

```diff
@@ -22,6 +22,7 @@ public void afterInvocation(IInvokedMethod method, ITestResult result) {
         result.setStatus(ITestResult.FAILURE);
         result.setThrowable(new TestFailedException());
       }
+      //todo transofrm result into better exception
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)